jaeger-crossdock/https-proxy/authority.key

```diff
@@ -0,0 +1,8 @@
+-----BEGIN EC PARAMETERS-----
+BggqhkjOPQMBBw==
+-----END EC PARAMETERS-----
+-----BEGIN EC PRIVATE KEY-----
+MHcCAQEEIAA24MB8sxrLG1an0nG1DCH6J32iqrtborxFOjdqWNCmoAoGCCqGSM49
+AwEHoUQDQgAEAhYnw9zYU0G3VZ48nNlT5jAs096pX0zeHM/yxiJe+DS5Yj0EJXM/
+0A1Of2zqxbyJpaEIFcqTmTTyTXCE7I6B6Q==
+-----END EC PRIVATE KEY-----
```

Comment:
are these being generated by `jaeger-crossdock/https-proxy/gen.sh` every time?

Reply:
No, this is root certificate which is used to generate proxy's certificate by `gen.sh`. Hard-coding is ugly by the way, but sometimes test project includes.

The Public key pin is coded here:
https://github.com/jaegertracing/jaeger-client-java/pull/602/files#diff-9d280e9b01ffb0fc60647fe5d2a8286cR74

Reply:
OK, I will.